_episodes/10-development-setup.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Development Setup"
```

Comment:
the number 10 in the filename doesn't correspond with the number (8) listed in the lesson overview page.

Reply:
10 is the number in the outline document. I think it is better to wait for episode 8 and 9 to be added instead of moving this episode